main_test.go

```diff
@@ -144,7 +144,20 @@ func TestFullScrapeCycle(t *testing.T) {
 
 	body, _ := ioutil.ReadAll(resp.Body)
 
-	expected := `# HELP kube_pod_info Information about pod.
+	// TODO: Move this to a golden record file?
```

Comment:
as I mentioned in #590 I don't think the maintenance burden of this is actually worth it, I'd vote for removing this check entirely, as the unit and e2e tests already validate the same, without the additional maintenance required